ORION/unit-test/generate-batch-test-data.py

```
@@ -330,6 +400,7 @@ def generate_batch_with_multiple_fingerprints(
     print(f"     - WITH rcos field: STABLE (no historical match, treated as new)")
     print(f"     - WITHOUT rcos field: REGRESSION (30% drop detected)")
     print(f"  ⚠️  Fingerprint E (threads=256): REGRESSION (busy_cpu doubled)")
+    print(f"  ⚠️  Fingerprint F (multibench): REGRESSION (composite -25%)")
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)